tests/PHPStan/Analyser/nsrt/bug-13197b.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types = 1);
+
+namespace Bug13197b;
+
+use function PHPStan\Testing\assertType;
+
+function execute(string $command): void
+{
+	if (!function_exists('proc_open')) {
+		return;
+	}
+
+	$pipes = [];
+
+	$process = @proc_open(
+		$command,
+		[
+			['pipe', 'rb'],
+			3 => ['pipe', 'wb'], // stdout
+			5 => ['pipe', 'wb'], // stderr
```

Comment:
```suggestion
			3 => ['pipe', 'wb'], // https://stackoverflow.com/questions/28909347/is-it-possible-to-connect-more-than-the-two-standard-streams-to-a-terminal-in-li#28909376
			5 => ['pipe', 'wb'],
```

small comment improvement, `stdout` is more or less always `1` pipe index